File: 1.two-sum/main.cpp
#include<iostream>
#include<vector>
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int>& num, int target) 
    {
        vector<int>out;
        for (int i = 0; i < num.size()-1; i++)
        {
            for (int k = i+1; k < num.size(); k++)
            {
                if (num[i] + num[k] == target)
                {
                    out.push_back(i);
                    out.push_back(k);
                    return out;
                }
            }
           
        }
        return out;
    }
};

File: 1.two-sum/main.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(0, len(nums)-1):
            for k in range(i+1, len(nums)):
                if nums[i] + nums[k] == target:
                    return [i, k]
                
    